Minutes 2021-05-06 (#218)
solid/authorization-panel: meetings/2021-05-26.md

# 2021-05-19 Solid Authorization

https://meet.jit.si/solid-authorization

## Agenda

* Minutes https://github.com/solid/authorization-panel/pull/214
* Use Case 1: Resource Access https://github.com/solid/authorization-panel/pull/215
* Use Case 3: Inheritance https://github.com/solid/authorization-panel/pull/216

### If time allows (from previous week)
* Inheritance with [Referenced Data Grants](https://solid.github.io/data-interoperability-panel/specification/#datamodel-referenced-data-grant) (interop panel)
  * UC [Conditional access by relationship](https://solid.github.io/authorization-panel/authorization-ucr/#conditional-relationship)

>   * Can we 'search and replace' Report -> Project and Measurement -> Task ? This will align it with scenario used in interop spec, primer and shape trees spec

Reviewer: ```suggestion
  * Can we 'search and replace' `Report` -> `Project` and `Measurement` -> `Task`? This will align it with scenario used in interop spec, primer, and shape trees spec
```

## Present

* E Pavlik
* Henry Story
* Matthieu B
* Justin B

## Minutes

Elf: Could we show best practice of how the statements are divided between resources?

> Justin: There are a number of use cases that rely on creating and deleting things. Are we satisfied with the notion of using WAC modes accross the board and if so are they insufficient or not to satisfy the use cases that we have? (Deletion/Creation)

Reviewer: ```suggestion
Justin: There are a number of use cases that rely on creating and deleting things. Are we satisfied with the notion of using WAC modes across the board, and if so, are they sufficient to satisfy the use cases that we have? (Deletion/Creation)
```

Matthieu: The idea is to use ACL modes and add new ones as needed.

> Justin: ACL modes as they stand might be insufficient for some use cases. Probably we could address the use cases and flag where the defficiency shows.

Reviewer: ```suggestion
Justin: ACL modes as they stand might be insufficient for some use cases. Probably we could address the use cases and flag where the deficiency shows.
```

Henry: We can try to see if it works with what we have and if not we can see if we need a new mode.

> Elf: How is deleting of an ACR supposed to work?

Reviewer: ```suggestion
Elf: How is deleting an ACR supposed to work?
```
